### PR TITLE
Passing testID to static header

### PR DESCRIPTION
**Pull request by Inbal-Tish**
to defer from calendar header arrows' testIDs

### CODE REVIEW

File: src/calendar/header/index.js [hunk 1]
```diff
@@ -79,14 +79,15 @@ class CalendarHeader extends Component {
...
+    const {testID} = this.props;
 
     if (!this.props.hideArrows) {
       leftArrow = (
         <TouchableOpacity
           onPress={this.onPressLeft}
           style={this.style.arrow}
           hitSlop={{left: 20, right: 20, top: 20, bottom: 20}}
-          testID={CHANGE_MONTH_LEFT_ARROW}
+          testID={testID ? `${this.testID}-${CHANGE_MONTH_LEFT_ARROW}`: CHANGE_MONTH_LEFT_ARROW}
```
Comment (ethanshar) [inline src/calendar/header/index.js]: is `this.testID` contain anything? why `this`?

File: src/calendar/header/index.js [hunk 2]
```diff
@@ -101,7 +102,7 @@ class CalendarHeader extends Component {
           onPress={this.onPressRight}
           style={this.style.arrow}
           hitSlop={{left: 20, right: 20, top: 20, bottom: 20}}
-          testID={CHANGE_MONTH_RIGHT_ARROW}
+          testID={testID ? `${this.testID}-${CHANGE_MONTH_RIGHT_ARROW}`: CHANGE_MONTH_RIGHT_ARROW}
```
Comment (ethanshar) [inline src/calendar/header/index.js]: same here..

File: src/calendar/header/index.js (hunk 1 above)
Comment (Inbal-Tish) [reply src/calendar/header/index.js]: Fixed.

File: src/calendar/header/index.js (hunk 2 above)
Comment (Inbal-Tish) [reply src/calendar/header/index.js]: Fixed.

**Pull request merged**